Impossible-Agent-Suite/gas-control-agent/README.md

```diff
@@ -0,0 +1,19 @@
+# High Gas Usage Agent
+
+## Description
+
+This agent detects transactions with high gas Price - above 10 Gwei
+
+## Supported Chains
+
+- BSC
+
+## Alerts
+
+- IMPOSSIBLE-2
+  - Fired when a transaction uses more than 10 Gwei as `Gas Price`
+  - Severity is always set to "High"
+  - Type is always set to "Info"
+  - Metadata includes:
+    - `contract`: The contract address that transaction interacted.
+    - `gas`: The gas price used.
```

Comment:
Are there any transactions on the BSC network that would trigger this agent that we can use as an on-chain example?
If there aren't then it shouldn't be a problem, but it would be nice to have.

Reply:
I took a look but could not find